apps/src/blockly/customBlocks/googleBlockly/proceduresBlocks.ts

```diff
@@ -143,13 +143,18 @@ GoogleBlockly.Extensions.register(
 GoogleBlockly.Extensions.register(
   'procedure_def_mini_toolbox',
   function (this: ProcedureBlock) {
-    // TODO: Add comment block here after https://codedotorg.atlassian.net/browse/CT-121
```

Comment:
We ended up adding a description field to the block rather than adding a comment block to the minitoolbox.